homeassistant/components/netgear/router.py

```diff
@@ -186,6 +197,13 @@ async def async_update_device_trackers(self, now=None) -> None:
 
         return new_device
 
+    async def async_get_traffic_meter(self) -> None:
+        """Get the traffic meter data of the router."""
+        with self._api_lock:
+            self.traffic_data = await self.hass.async_add_executor_job(
```

Comment:
I'd return the traffic data here. Then we don't need the router attribute. The data will be cached on the coordinator anyway.

Reply:
sure will change it